mep/books/templates/books/work_cardlist.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static mep_tags %}
+
+{% block page-subtitle %}{{ work.title }} Cards · Library Holdings · {% endblock %}
+
+{% block js %}
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=work.title style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+{# breadcrumbs & tabs #}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="tabs">
+    <li class="tab">
+        <a href="{% url 'books:book-detail' slug=work.slug %}">bibliography</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'books:book-circ' slug=work.slug %}">circulation</a>
+    </li>
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'books:book-card-list' slug=work.slug %}">cards</a>
+    </li>
+</nav>
+
+<section class="book-cards">
+    {% if footnotes %}
+    <ol class="card-results">
+    {% for footnote in footnotes %}
+    <li class="card result">
+        {% with card=footnote.image member=footnote.content_object.account.persons.first %}
+        <a href="{% url 'people:member-card-detail' member.slug card.short_id %}#{{ work.slug }}_{{ footnote.content_object.partial_start_date }}">
+                {# card thumbnails with member name and event year #}
+                <picture>
+                  {% with card.image|iiif_image:"size:width=225" as 1xthumbnail %}
+                    <source srcset="{{ 1xthumbnail }}, {{ card.image|iiif_image:"size:width=450" }} 2x">
+                    <img src="{{ 1xthumbnail }}" alt="" loading="lazy">
+                  {% endwith %}
+                </picture>
+
+                <div class="name">
+                    <dt>Card Holder</dt>
```

Comment:
`<dt>` and `<dd>` are only legal as children of a `<dl>`, but I don't see an enclosing `<dl>` here. if the intent is to semantically associate them with the card image, maybe we could use `aria-describedby`, since the image has no alt currently?

Reply:
Huh, whoops. I adapted from the card list result template, which is structured the same way (but no longer used).  Should I switch to `<dl>` ? 

We don't have an alt, but actually in this context maybe we could, e.g.: `lending library card for <member> ` (although should probably incorporate the date too).

If we use an `aria-describedby` what would it point to? Convert to `<dl>` and reference it?

Reply:
oh, I see now that the card result snippet has an `aria-label` with the member name. I could add something like that

Reply:
adding an `aria-label` with the name sounds good to me. I don't think we need a `<dl>` here; they can just be `<span>`s (but whatever you decide is fine).